main/src/ca/uwaterloo/flix/api/lsp/provider/completion/KeywordCompleter.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2023 Lukas Rønn
```

Comment:
Since this code is moved from CompletionProvider, we should also retain the copyright information from that file.